### PR TITLE
docs: add more details to the contribution guide

### PR DESCRIPTION
**Pull request by Keenuts**
Give some info around review SLO, and how we distribute reviews around.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,25 +77,28 @@ requested:
...
 We try to review PRs at least once per 24h window.
 If your PR remains untouched for more than 24h, please ping us.
 Anything will do "PTAL", "up".
 Sometimes, we are on holiday, but you cannot guess, so in doubt, ping us 😊.
 
-Maintainers don't own specific parts of the project. We try to use our best
-judgment when doing reviews.
-For small PRs (trivial fix, or additions), we consider 1 reviewer is enough,
-and can merge the PR directly after the review.
+When a maintainers sees an unassigned PR, they have 2 options:
```
Comment (apazylbe) [inline CONTRIBUTING.md:94]: s/maintainters/maintainer

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,25 +77,28 @@ requested:
...
-approve themselves.
- - if a maintainer requests changes without approval, they want to look at
-it again once the changes are in.
+When a change has multiple reviewers, we try to follow the following rule:
+- If you are the first to approve, you delegate merging to the other(s).
+- If you are the last to approve, it's your responsibility to merge.
+- If you request changes without approval, others must wait for your approval
+before merging (and if you are the last to approve, you have to merge).
+
+### Who merges
```
Comment (apazylbe) [inline CONTRIBUTING.md:108]: Should this heading be above?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -77,25 +77,28 @@ requested:
...
+- Otherwise, they will chose 2 or more reviewers (can self-assign).
+- If a maintainer wants to review, they can add themselves to the reviewer list.
+- If a maintainer comments without adding a review, they don't block merging.
 
-Between maintainers, we try to follow the following convention:
- - if a maintainer approves a PR, the others are free to merge if they
-approve themselves.
- - if a maintainer requests changes without approval, they want to look at
-it again once the changes are in.
+When a change has multiple reviewers, we try to follow the following rule:
```
Comment (apazylbe) [inline CONTRIBUTING.md:102]: I think this means reviewers do the merge, and all reviewers must approve?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -77,25 +77,28 @@ requested:
...
-judgment when doing reviews.
-For small PRs (trivial fix, or additions), we consider 1 reviewer is enough,
-and can merge the PR directly after the review.
+When a maintainers sees an unassigned PR, they have 2 options:
 
-For larger PRs and non-trivial changes, 2 pair of eyes are always good.
+- If it's a small & trivial change, they will self-assign, and handle
+the approval/merging.
+- Otherwise, they will chose 2 or more reviewers (can self-assign).
+- If a maintainer wants to review, they can add themselves to the reviewer list.
```
Comment (apazylbe) [inline CONTRIBUTING.md:99]: Super nit, but I think the formatting makes it sounds like 4 options, and these two are still options

File: CONTRIBUTING.md (hunk 3 above)
Comment (Keenuts) [reply CONTRIBUTING.md]: Yes, rephrased a bit to make it clear :)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -68,13 +70,45 @@ requested:
...
+Sometimes, we are on holiday, but you cannot guess, so in doubt, ping us 😊.
+
+When a maintainer sees an unassigned PR, they have 2 options:
+
+- If it's a small & trivial change, they will self-assign, and handle
+the approval/merging.
+- Otherwise, they will choose 2 or more reviewers (can self-assign).
+
+When a change has multiple reviewers, we try to follow the following rules:
+- All reviewers must approve.
```
[the remark this answers is not in the document]
Comment (Keenuts) [reply CONTRIBUTING.md:102]: If the change is trivial, we fallback to the first case where we have only 1 reviewer no? Or is that unclear?

**Pull request merged**